working_copy/machinev1

```diff
@@ -284,7 +284,7 @@
     },
     {
       "description": "Besides a local abuse of data and systems the information security can be endangered by a successful account or application compromise. Furthermore attacks are possible that intercept and access information during transmission (wiretapping, spoofing or hijacking). Human/configuration/software error can also be the cause.",
-      "expanded": "Information Content Security",
+      "expanded": "Information Security",
       "value": "information-content-security"
```

Comment:
We would need also to update the value of the machine tag.

Reply:
The machine tag was kept on purpose, as that would be a breaking change. It can be noted on #118 to update it on next major version.

Reply:
I added it to the list in #118